ARCs/arc-0086.md

```diff
@@ -0,0 +1,149 @@
+- Total committee members (`uint64`), is the size of the selected list; 
+- Total committee votes (`uint64`), is the sum of votes in the selected list.
+
+The xGov Committee selection is repeated periodically to select new xGov Committees over time.
+
+### xGov Committee Selection Procedure
+
+Given the xGov Committee parameters `(I, Bc, Bi, Bf)`, the selection is executed with the following procedure:
+
+1. Collect all proposed blocks in the range `[Bi; Bf]` to build the `potential_committee` (note that not all the Block Proposers are eligible as xGov). For each Block Proposer address in the `potential_committee`, assign a voting power equal to the number of blocks proposed in the range `[Bi; Bf]`.
+
+2. Collect all the _eligible_ xGovs in the range `[Bc + I; Bf]` to build the `eligible_xgovs` list by:
+    - Adding to the list all xGov with a _subscription_ acknowledgements on the xGov Registry in the range;
+    - Removing from the list all xGov with a _unsubscription_ acknowledgements on the xGov Registry in the range;
+
+3. Filter `potential_committee` ∩ `eligible_xgovs` to obtain the final `committee`.
+
+### Representation
+
+The xGov Committee selection **MUST** result in a JSON with following schema:
+
+```json
+{
+  "title": "xGov Committee",
+  "description": "Selected xGov Committee with voting power and validity",
+  "type": "object",
+  "properties": {
+    "xGovs": {
+      "description": "xGovs with voting power, sorted lexicographically with respect to addresses",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "address": {
+            "description": "xGov address used on xGov Registry",
+            "type": "string"
+          },
+          "votes": {
+            "description": "xGov voting power",
+            "type": "number"
+          }
+        },
+        "required": ["address", "votes"]
+      },
+      "uniqueItems": true
+    },
+    "inceptionPeriod": {
+      "description": "Number of blocks after the xGov Registry creation block, reserved to the first xGov onboarding",
+      "type": "number"
+    },
+    "registryCreation": {
+      "description": "The xGov Registry creation block",
+      "type": "number"
+    },
+    "periodStart": {
+      "description": "First block of the Committee selection period, must ≡ 0 mod 1,000,000 and greater than registryCreation + inceptionPeriod",
+      "type": "number"
+    },
+    "periodEnd": {
+      "description": "Last block of the Committee selection period, must ≡ 0 mod 1,000,000 and greater than periodStart",
+      "type": "number"
+    },
+    "members": {
+      "description": "Total number of Committee members",
+      "type": "number"
+    },
+    "votes": {
+      "description": "Total number of Committee votes",
+      "type": "number"
+    }
+  },
+  "required": ["xGovs", "periodStart", "periodEnd", "members", "votes"],
```

Comment:
```suggestion
  "required": ["networkGenesisHash", "periodEnd", "periodStart", "registryId", "totalMembers", "totalVotes", "xGovs"],
```

Reply:
All the fields are required (I’ve sorted them lexicographically here for convenience).